### PR TITLE
fix: exports from packages, add guide for making derived SDKs

### PR DESCRIPTION
**Pull request by haved**
Also adds some exports to core and stable, so there are some `feat` commits. (You could change them if you don't want a MINOR bump). Fixes casing in an api class (should not a breaking change, as the type wasn't exported).

### CODE REVIEW

**Comment by haved** [comment]
I was going to slack to joke about needing a review by tomorrow before I lose access, but it's already gone :(

File: guides/derived_SDKs.md [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+}
+
+export class CoolThingAPI extends BaseResource<CoolThing> {
+
+    /**
+    * [Create cool things](https://doc.cognitedata.com/api/v1/#operation/postCoolThing)
+    *
+    * ```js
+    * const coolthings = [
+    *   { name: 'Shrub', coolness: 40 },
```
Comment (polomani) [inline guides/derived_SDKs.md:118]: 🤣

**Review by polomani — APPROVED** [review]
looks cooler then cactus to me!

**Comment by polomani** [comment]
ps: we might need to squash it to `fix:` or something, since it's not just a documentation

**Review by hhenriques — APPROVED** [review]
Great write up!

**Review by mrtraser — APPROVED** [review]
cool way to explain things, well done!

**Pull request merged**